### ISSUE TITLE
url_launcher won't work on IOS simulator. fatal error: 'url_launcher/UrlLauncherPlugin.h' file not found

### ISSUE THREAD

**Issue by Sirrr** [flutter/flutter#27893]
Dears, I am struggling with this issue. Please help.
```yml
url_launcher: ^5.0.1
cocoapods: 1.6.0
```
```
Launching lib/main.dart on iPhone X in debug mode...
Warning: CocoaPods not installed. Skipping pod install.
CocoaPods is used to retrieve the iOS platform side's plugin code that responds to your plugin usage on the Dart side.
Without resolving iOS dependencies with CocoaPods, plugins will not work on iOS.
For more info, see https://flutter.io/platform-plugins
To install:
brew install cocoapods
pod setup

Starting Xcode build...
Xcode build done. 5.0s
Failed to build iOS app
Error output from Xcode build:
↳
** BUILD FAILED **

Xcode's output:
↳
=== BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Debug ===
Debug.xcconfig line 1: Unable to find included file "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
Debug.xcconfig line 1: Unable to find included file "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
=== BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Debug ===
/Users/siraj/dev/AndroidStudioProjects/delete/ios/Runner/GeneratedPluginRegistrant.m:6:9: fatal error: 'url_launcher/UrlLauncherPlugin.h' file not found
#import <url_launcher/UrlLauncherPlugin.h>
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

Could not build the application for the simulator.
Error launching application on iPhone X.
```

# Flutter doctor
```
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.14 18A391, locale en-AZ)
• Flutter version 1.0.0 at /Applications/flutter
• Framework revision [5391447](https://github.com/flutter/flutter/commit/5391447fae6209bb21a89e6a5a6583cac1af9b4b) (3 months ago), 2018-11-29 19:41:26 -0800
• Engine revision [7375a0f](https://github.com/flutter/flutter/commit/7375a0f414bde4bc941e623482221db2fc8c4ab5)
• Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
• Android SDK at /Users/siraj/Library/Android/sdk
• Android NDK location not configured (optional; useful for native profiling support)
• Platform android-28, build-tools 28.0.3
• Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
• Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
• All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
• Xcode at /Applications/Xcode.app/Contents/Developer
• Xcode 10.1, Build version 10B61
• ios-deploy 1.9.4
• CocoaPods version 1.6.0

[✓] Android Studio (version 3.3)
• Android Studio at /Applications/Android Studio.app/Contents
• Flutter plugin version 32.0.1
• Dart plugin version 182.5215
• Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[!] IntelliJ IDEA Ultimate Edition (version 2017.2.5)
• IntelliJ at /Applications/IntelliJ IDEA.app
✗ Flutter plugin not installed; this adds Flutter specific functionality.
✗ Dart plugin not installed; this adds Dart specific functionality.
• For information about installing plugins, see
https://flutter.io/intellij-setup/#installing-the-plugins

[✓] VS Code (version 1.30.2)
• VS Code at /Applications/Visual Studio Code.app/Contents
• Flutter extension version 2.22.3

[✓] Connected device (1 available)
• iPhone X • A8EB16AF-4992-4C41-ADC2-7A637B0ABFA2 • ios • iOS 12.1 (simulator)
```

**Comment by zoechi** [comment]
@Sirrr please try https://github.com/flutter/flutter/issues/16049#issuecomment-382629492

Similar to #15409 (shared_preferences)

**Comment by Sirrr** [comment]
@zoechi Thank you for the direction. I managed to solve the problem through running 'flutter run' in the terminal.

At first I changed the Podfile as described in the comment. But then I realized it works without changes, by just calling 'flutter run' in the terminal. I am using Android Studio to build project. But, I guess it does not run 'pod install' when it builds the app. So the missing Pod packages were causing the issue.
